### ISSUE TITLE
Create and open a Post-Vote menu

### ISSUE THREAD

**Issue by evandrocoan** [evandrocoan/MultiModServer#40]
### Create and open a Post-Vote menu

After the player vote, open a new menu showing the options:

1. See the voting progress
1. Undo my vote
1. Close the vote menu

If the user choose `See the voting progress` still allow him to see the other new menu at the `0 - None` option by:

1. https://github.com/evandrocoan/Galileo/issues/4 Allowing to close the vote menu when the vote status is 3 - always show the vote current progress

Basically this is the same menu opened the new menu above.

**Comment by evandrocoan** [comment]
Implemented:

```pawn
// Shows the voting option "None" for people to choose if they don't
// like any other choice.
//
// 0 - Disabled this feature.
// 1 - Enabled this feature.
// 2 - Instead of adding a new menu option as `None`, to add a new
//     submenu allowing to choose the option: `None`, `Go back`,
//     `Close The Voting Menu` and `Undo My vote`.
gal_vote_show_none 2
```

![image](https://cloud.githubusercontent.com/assets/5332158/21467826/b1bdbdda-c9e1-11e6-9b31-e0f11b563b71.png)

![image](https://cloud.githubusercontent.com/assets/5332158/21467819/9d8ed13c-c9e1-11e6-926a-e27152bd1786.png)
